episodes/02-filedir.md

```diff
@@ -821,6 +821,18 @@ Library/              Users/
 Network/              Volumes/
 ```
 
+
+:::::::::::::::::::::::::::::::::::::::::  callout
+### When to use short or long options
+When options exist as both short and long options:
+
+- Use the short option when typing commands directly into bash to minimise typing.
+  This enables you to get your task done faster.
+- Use the long option in scripts to provide clarity.
+  It will be read many times and typed ones.
```

Comment:
```suggestion
- Use the short option when typing commands directly into
the shell to minimize keystrokes and get your task done faster.
- Use the long option in scripts to provide clarity.
  It will be read many times and typed once.
```